common/transport.ts

```diff
@@ -64,7 +72,12 @@ export class Transport {
     this.apiKmsKey = apiKmsKey;
   }
 
-  applyEnvVars(lambdas: lambda.Function[]) {
+  /*
+   * This function's parameter/return type changed.
+   *
+   * ILambdaFunction is used instead of a AWS import
+   */
+  applyEnvVars(lambdas: ILambdaFunction[]) {
```

Comment:
Unchanged aside from `lambdas` parameter type (from aws import to LambdaFunction interface)